plugins/azure-devops/api-report.md

```diff
@@ -69,6 +69,7 @@ export interface AzureDevOpsApi {
   // (undocumented)
   getBuildRuns(
     projectName: string,
+    entityRef: string,
```

Comment:
Why we need the `entityRef`? It's been a bit since I've setup new permissions myself, I'm pretty sure this is needed to allow for more fine-grain permissions but would like clarification, please.

Reply:
Since the permissions are resource-based (`RESOURCE_TYPE_CATALOG_ENTITY`). The adopter can use the conditional decisions such as (`isEntityOwner or hasLabels or...`) exposed by the catalog by default, plus with their own set of policies in `backend/src/plugins/permission.ts`. 

So the `entityRef` has been passed to the backend call to perform the permission decision. 

An option, In future we can plan to move annotation parsing to the backend and get all the required information from the `catalogApi`.

Reply:
Great, that helps me remember why this is needed. Thanks!